generators/entity-client/templates/angularjs/src/main/webapp/app/entities/_entity-management-detail.html

```diff
@@ -26,7 +26,7 @@ <h2><span data-translate="<%= keyPrefix %>.detail.title"><%= entityClassHumanize
             const fieldName = fields[idx].fieldName;
             const fieldType = fields[idx].fieldType;
             const fieldTypeBlobContent = fields[idx].fieldTypeBlobContent; _%>
-        <dt><span data-translate="<%= keyPrefix %>.<%= fieldName %>"><%= fields[idx].fieldNameHumanized %></span></dt>
+        <dt><span data-translate="<%= keyPrefix %>.<%= fieldName %>" <%_ if (fields[idx].javadoc) { _%> uib-tooltip="{{'<%= keyPrefix %>.help.<%= fieldName %>' | translate}}" <%_ } _%>><%= fields[idx].fieldNameHumanized %></span></dt>
```

Comment:
I wouldnt bother with angularJS as we are planning to drop it

Reply:
I know but it's quick to do it, so I do it and it will be integrate in the external generator in this way.